crates/tauri-runtime-wry/src/lib.rs

```diff
@@ -530,7 +530,7 @@ impl<'a> From<&TaoWindowEvent<'a>> for WindowEventWrapper {
         scale_factor: *scale_factor,
         new_inner_size: PhysicalSizeWrapper(**new_inner_size).into(),
       },
-      #[cfg(any(target_os = "linux", target_os = "macos"))]
+      #[cfg(any(target_os = "linux", target_os = "macos", target_os = "windows"))]
```

Comment:
```suggestion
```

This function won't be used on mobile afaik so we might as well remove this cfg flag